### PR TITLE
Fixed the checking of rpmrebuild

### PR DESCRIPTION
**Pull request by damianam**
https://github.com/hpcugent/easybuild-framework/issues/1587#issuecomment-194744938

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyblocks/generic/rpm.py [hunk 1]
```diff
@@ -45,6 +45,7 @@
 from easybuild.easyblocks.generic.binary import Binary
 from easybuild.framework.easyconfig import CUSTOM
 from easybuild.tools.build_log import EasyBuildError
+from easybuild.tools.filetools import which
 from easybuild.tools.run import run_cmd
 from easybuild.tools.systemtools import check_os_dependency
```
Comment (boegel) [inline easybuild/easyblocks/generic/rpm.py]: drop this import?

Comment (damianam) [inline easybuild/easyblocks/generic/rpm.py]: It is needed here in line 125. But I think that one can be also replaced by `which`. Would you agree?

Comment (boegel) [inline easybuild/easyblocks/generic/rpm.py]: Yes, I see no reason not to replace that too.

**Comment by boegel** [comment]
lgtm (and retested), going in, thanks @damianam!

**Pull request merged**